### PR TITLE
Css positioning - Tev Nicolas

### PR DESCRIPTION
**Pull request by tevnicolas**
Screenshot 2024-02-08 at 4 46 45 PM
Screenshot 2024-02-08 at 5 08 20 PM
Screenshot 2024-02-08 at 7 10 09 PM

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  /* your code here */
+  position: absolute;
+  bottom: 0;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+
+  /* your code here */
+}
```
Comment (shawnkost) [inline css-positioning/absolute.css:47]: Missing code for the `.overlay` to get it to appear

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Tev!

File: css-positioning/absolute.css (hunk 1 above)
Comment (tevnicolas) [reply css-positioning/absolute.css:47]: Oh okay, I just checked and I had a linear gradient appearing, but the code for it is under the .meme rule set. I'll change it now to appear under .overlay.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**